src/openms/source/COMPARISON/SPECTRA/SpectraSTSimilarityScore.cpp

```diff
@@ -95,6 +95,7 @@ namespace OpenMS
     double min_high_intensity = 0;
     if (!spec.empty())
     {
+      //@todo timosachsenberg Is this a dirty undocumented hack or just wrong?
```

Comment:
@timosachsenberg Can you have a look at that? Since cut_peaks_below is a UInt, I think this will always be either inf, 0 or 1

Reply:
Yes this looks wrong. Should be doubles (and probably not use any sorting - just pick max)